### ISSUE TITLE
Evaluation on MOW dataset

### ISSUE THREAD

**Issue by dqj5182** [janehwu/mcc-ho#3]
For evaluating on MOW dataset, did you use the raw 3D object mesh from MOW dataset as GT 3D object mesh? Or did you use watertight 3D object mesh as GT 3D object mesh?

I am aware that you have followed the evaluation protocol of "What's in your hand" paper. But they do not provide exact detail regarding the evaluation on MOW dataset. And if you used the raw 3D object mesh, I am curious how you handled problems from their raw mesh such as self-intersection and inverted faces.

**Comment by janehwu** [comment]
Hi! We used the ground truth 3D object mesh from the MOW dataset. Meaning, we did the following:

1) Download the object meshes according to the MOW repo: https://github.com/ZheC/MOW.
2) Read `poses.json` in the MOW repo to transform each object mesh to the correct ground truth 6DoF pose + scale.

We did not check that each object mesh is watertight (which isn't necessary for the 3D metrics computed), but I would assume most of the CAD models are watertight.
